src/astt_gui/app.py

```diff
@@ -1,5 +1,7 @@
-"""Facilitate integration between the GUI and system components."""
+# pylint: disable= W0603, W0718, W0602, E0401
+# mypy: disable_error_code="import-untyped"
```

Comment:
Should not be ignored but rather addressed, Maybe in the decoupling ticket.

Reply:
I agree. We should pick up the work with @ ssithole1 ASAP.